Moodlet/app/src/main/java/com/cmput3owo1/moodlet/activities/MainActivity.java

```diff
@@ -3,6 +3,7 @@
 import android.os.Bundle;
 
 import com.cmput3owo1.moodlet.R;
+import com.cmput3owo1.moodlet.fragments.RegisterFragment;
```

Comment:
is this import necessary? There's no other changes in this file

Reply:
removed